### ISSUE TITLE
[Resources & Support] [Drupal] Fix instances of broken ARIA references in CMS

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-cms#22745]
## Describe the defect
Within the edit screens on the Resources & Support content type, there are multiple `aria-describedby` attributes that do not correspond to an ID anywhere on the page.
  - Page Introduction
  - Calls to Action
  - Main content
  - VA Benefits hubs
  - Related Information
  - Other Categories where people may look for this article

## Tasks from Jacqueline
Jackie reviewed ticket #15822 and has provided the following actions and AC

- The 'Page Introduction' field (in the 'Title and introduction' section) has an `aria-describedby="edit-field-intro-text-limited-html-0--description"` on a wrapping div that doesn't match an `id` on the page. The description text in that field, `div#edit-field-intro-text-limited-html-0-value--description`, is already linked to the textarea input with `aria-describedby`, so it does not need to be associated to the wrapping div.
  - [ ] Remove the `aria-describedby="edit-field-intro-text-limited-html-0--description"` on the wrapping `<div>`.
 
- The 'Calls to Action' block has an `aria-describedby="edit-field-buttons--description"` on a wrapping div that doesn't match an `id` on the page. The table nested in that wrapping div also has `aria-describedby="edit-field-buttons--description"`.  The description is already nested inside of the table header and inside of the h4 with the heading text, so it doesn't need to be linked using aria-describedby.
  - [ ] Remove the `aria-describedby` on the wrapping `<div>`.
  - [ ] Remove the `aria-describedby` on the `<table#field-buttons-values>`.

- The 'Main content' block has an `aria-describedby=edit-field-content-block--description"` on a wrapping div that doesn't match an `id` on the page. The table nested in that wrapping div also has `aria-describedby="edit-field-content-block--description"`.  The description is already nested inside of the table header and inside of the h4 with the heading text, so it doesn't need to be linked using aria-describedby.
  - [ ] Remove the `aria-describedby` on the wrapping `<div>`.
  - [ ] Remove the `aria-describedby` on the `<table>`.

- ~~The `<details>` tag of the 'VA Benefits hubs' block has `aria-describedby="edit-field-related-benefit-hubs--description"`, but if I search for that ID I can't find it anywhere on the page, it doesn't exist. There is a description field in the details component, but that info can be accessed by opening the details and reading as usual, so [using aria is unnecessary](https://dsva.slack.com/archives/C8E985R32/p1761168017435559).~~
  - [ ] ~~Remove the `aria-describedby` on the `<details>` tag.~~
  - [ ] This will be handled when site is updated to Drupal 11: https://www.drupal.org/project/drupal/issues/2896169

- The 'Related Information' block has an `aria-describedby=edit-field-related-information--description"` on a wrapping div that doesn't match an `id` on the page. The table nested in that wrapping div also has `aria-describedby="edit-field-related-information--description"`.  The description is already nested inside of the table header and inside of the h4 with the heading text, so it doesn't need to be linked using aria-describedby.
  - [ ] Remove the `aria-describedby` on the wrapping `<div>`.
  - [ ] Remove the `aria-describedby` on the `<table>`.

- ~~The `<details>` tag of the 'Other Categories where people may look for this article' dropdown (in the 'Article categories' section)  has `aria-describedby="edit-group-other-categories--description"`, but if I search for that ID I can't find it anywhere on the page, it doesn't exist. There is a description field in the details component, but that info can be accessed by opening the details and reading as usual, so [using aria is unnecessary](https://dsva.slack.com/archives/C8E985R32/p1761168017435559).~~
  - [ ] ~~Remove the `aria-describedby` on the `<details>` tag.~~
  - [ ] This will be handled when site is updated to Drupal 11: https://www.drupal.org/project/drupal/issues/2896169

### AC
The following fields/sections no longer throw errors about unlinked `aria-describedby` attributes:
- [ ] Page Introduction
- [ ] Calls to Action
- [ ] Main content
- [ ] ~~VA Benefits hubs~~
- [ ] Related Information
- [ ] ~~Other Categories where people may look for this article~~

## Steps to find Resources and support
1. Log into staging.cms.va.gov
2. Under Subsections, click the dropdown for VHA
3. In the list, click VHA Resources and support
4. Click Edit for any of the items in the list (but remember not to actually save or publish edits to existing content)

## Screenshots
Screenshot of the Page Introduction field error from AMP plugin. The `aria-describedby` attribute is pointing to something with the ID `edit-field-intro-text-limited-html-0--description` but if I search for that ID I can't find it anywhere on the page, it doesn't exist. 

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/6e5a871b-d1f8-4f9a-bcfb-f94eb48a4130)

Screenshot of where the attribute is on the `<div>`
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/74ccfcb4-2faa-4ddc-bb71-a8ec52647a92)

## Accessibility Standard
WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

### Roles / assignments
_After functional testing, code review, accessibility review, and design review can happen in parallel._
- [ ] **Functional testing:** 
- [ ] **Code review:** 
- [ ] **Design review:** 
- [ ] **Accessibility review:** @qjac 
- [ ] **Product acceptance:**

**Comment by dsasser** [comment]
## Status update 11/20/25

__[Related Facilities changes](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/22593) should resolve the issues in this ticket, in theory_

Today I met with @Becapa who is working on similar a11y [issues](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/22593) for Facilities. Once [this](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/22758) PR is ready for review (expected by end of week), we can test these issues there. This ticket, then, could be a no-op if we are correct.
